### ISSUE TITLE
Cannot compile on Windows 10.

### ISSUE THREAD

**Issue by Dermiste** [bminortx/Buckshot#9]
Hi,

I cannot seem to compile on Windows 10. BuckshotCreator.m and createMeshFile.sh seem to be meant for bash on linux, is there a way around it? or should I just try to run on OSX or linux? Thanks you

**Comment by bminortx** [comment]
I have never compiled on Windows, myself :( Don't have the software around. However, I will say that the createMexFiles.sh file is pretty simplistic; all it does is run CMake for you in the right directory. You could probably run the CMake command for yourself from the Command Prompt. I'll make a Windows-specific setup script when I have the software to test it :) Or you can give it a shot yourself and submit a PR!

**Comment by Dermiste** [comment]
True, that could also be an option. Also it depends on what is possible to be done through Buckshot. Ultimately I want to be able to control 3D physic simulation through simulink, but it appears that this Matlab tool is only an interface to generate simulation, not to control them, is that correct?

**Comment by bminortx** [comment]
Correct; any user input after the simulation begins is currently to the GUI alone, not the physics simulation settings themselves. I don't think there would be a reason this couldn't happen, though, programmatically-speaking. Good thought.
